### PR TITLE
DX-805 get and show pod connection details for livebook smart cell

### PR DESCRIPTION
**Pull request by Fadhil**
## Summary

In this PR, we update the kubernetes_pod query to additionally get the `node_name` and `cookie` for the pod and display those values along with a short instruction on how the user can connect to a selected node.

Ticket: [DX-805](https://mindvalley.atlassian.net/browse/DX-805)

The changes in action:

Screenshot 2024-10-01 at 3 35 16 PM

Testing out the provided details in a livebook:
Screenshot 2024-09-30 at 6 58 57 PM

## What's Changed

Show pod connection details and instructions on how to use them in the `application instances connect` flow

Updated schema and kubernetes_pods.graphql to get `node_name` and `cookie` 

[DX-805]: https://mindvalley.atlassian.net/browse/DX-805?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/228#" title="Author self-approved">Fadhil</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cli/src/commands/application/instances/connect.rs
Comment (jk-gan) [inline cli/src/commands/application/instances/connect.rs:285]: I'm think what about making it more structure? like 

```
To connect to mv-wukong-api-proxy:
1. start a new Notebook
2. click on `+Smart`
3. select `Remote Execution`. 
4. Follow the instructions that pop up to restart the Notebook.
5. Enter these values into the `NODE` and `COOKIE` input boxes:
    NODE: 🖥️: ...
    COOKIE: 🍪: ...
```

Comment (Fadhil) [reply cli/src/commands/application/instances/connect.rs:285]: I like this. So much clearer. Lemme update.

**Review by jk-gan — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/228#" title="Author self-approved">Fadhil</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/228#pullrequestreview-2341844870" title="Approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
